bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/library/types/StringListType.java

```diff
@@ -37,27 +36,25 @@ public StringListType() {
         typeDetails = Collections.emptyList();
     }
 
+    public StringListType(List<String> rows) {
+        this.typeDetails = new ArrayList<>(rows);
+    }
+
     public StringListType(StringType... rows) {
-        typeDetails = new ArrayList<String>(rows.length);
-        for (StringType row : rows) {
-            typeDetails.add(row.toString());
-        }
+        this(Arrays.stream(rows)
```

Comment:
Please avoid creating a collection twice, i.e. assign the result of the collector directly to `typeDetails` instead of passing it to the other constructor.